src/DssAutoLine.sol

```diff
@@ -26,7 +25,8 @@ contract DssAutoLine {
     /*** Events ***/
     event Rely(address indexed usr);
     event Deny(address indexed usr);
-    event File(bytes32 indexed ilk, bytes32 indexed what, uint256 data);
+    event Enable(bytes32 ilk, uint256 line, uint256 gap, uint256 ttl);
```

Comment:
One more thing, can we make ilk in both events `indexed`?

Reply:
added

Reply:
Thanks!